ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -81,7 +82,7 @@ public void write(final OutputStream out) {
             disconnectCheckInterval,
             TimeUnit.MILLISECONDS
         );
-        if (row != null) {
+        if (row != null && (row.key() != null || row.value() != null)) {
```

Comment:
Just curious, is this required?  It seems like you're just sending a null row, not null parts.